Automattic/wp-calypso
# Customer home: add section

#### Changes proposed in this Pull Request

* introduces the section for the upcoming Customer Home

<img width="1164" alt="Screen Shot 2019-08-07 at 18 41 30" src="https://user-images.githubusercontent.com/1041600/62660026-55815f00-b943-11e9-8ce3-9580f817bf3d.png">

* if the user can't see the Customer Home, they’ll see a message

<img width="1109" alt="Screen Shot 2019-08-08 at 14 38 58" src="https://user-images.githubusercontent.com/1041600/62726490-b6657180-b9ed-11e9-89c8-9c301898a06c.png">

#### Testing instructions

* manually type `/home/%site slug%` and `/home`
* both paths should work
* links should go to the correct section
* ensure it's only visible on development. Staging and wpcalypso shouldn't display this.
